build.sbt

```
@@ -83,6 +84,19 @@ lazy val rdfProtos = (project in file("rdf-protos"))
     publishArtifact := false,
   )
 
+// Intermediate project that generates the Scala code from the protobuf files
+lazy val rdfProtosJava = (project in file("rdf-protos-java"))
+  .enablePlugins(ProtobufPlugin)
+  .settings(
+    name := "jelly-javameta",
```

```suggestion
    name := "jelly-protos-java",
```

the scalameta here referred to a scala lib for metaprogramming

---

doned